ConsentViewController/Classes/SourcePointClient/Adapters.swift

```diff
@@ -501,6 +501,14 @@ extension SourcepointClientCoordinator.State {
             ccpa: self.ccpa.toCore(metaData: self.ccpaMetaData),
             usNat: self.usnat.toCore(metaData: self.usNatMetaData),
             ios14: self.ios14.toCore(),
+            preferences: State.PreferencesState(
+                metaData: State.PreferencesStatePreferencesMetaData(
+                    configurationId: "",
+                    additionsChangeDate: InstantKt.toInstant("2025-03-25T18:38:50.090Z"),
+                    legalDocLiveDate: nil
```

Comment:
@andresilveirah, please pay attention to correctly resolving this in case of a merge conflict